### ISSUE TITLE
Errors in Coale & Demeny functions

### ISSUE THREAD

**Issue by hanase** [hanase/demogR#3]
From Jim Oeppen:

1. The model inputs include vectors of e10f  and e10m. These should be Region-specific,but demogr uses the female and male West series for all regions.
2. Many of the parameters are Region- and sex-specific, but demogr often uses the West female in the wrong context.
3. The code does not correct for the printing errors in the Coale & Demeny volumes identified by Gough (1987) in Population Index 53, 1, pp. 3-4.
4. There are at least 6 typing errors in the ax, bx, ax1, and bx1 parameters.

See also #4

**Comment by hanase** [comment]
From Jim Oeppen:

I estimated the etenf and etenm parameters for each region and sex using univariate R:optimize to minimise the sum of squared differences at each level between my input qx tables typed from C&D (1983) and the qx calculated by the sex- and region-specific function.  (I did not use a specialised sum-of-squares algorithm.)  Obviously, this assumes no typing errors in my input qx tables.  I have inspected them as 3D surfaces and there are no obvious spikes or holes.  My etenm and etenf estimates for model West are very similar to the demogr values.

However, if I plot the error surfaces after the optimization there is a systematic pattern of small errors for ages 75+.  The maximum absolute error is about 0.06 per thousand, so the errors are very small.  The patterns are similar across all Regions, but each sex has its own pattern.  It may be a programming error by me, but I cannot find the source for this.  As the errors below age 75 are generally random and centred on zero, I wonder if the formulae for mu(77.5) and mu(105) were not given with enough decimal precision by C&D.  These formulae are sex-specific, but common to all Regions, which fits with the observed error patterns.
